apps/api/src/app/blueprint/e2e/create-blueprint-notification-templates.e2e.ts

```diff
@@ -1,11 +1,12 @@
 import { expect } from 'chai';
+
 import { UserSession } from '@novu/testing';
 import { EmailBlockTypeEnum, StepTypeEnum, INotificationTemplate, FilterPartTypeEnum } from '@novu/shared';
 import { NotificationTemplateRepository, EnvironmentRepository } from '@novu/dal';
 
-import { CreateNotificationTemplateRequestDto } from '../dto';
+import { CreateNotificationTemplateRequestDto } from '../../notification-template/dto';
 
-describe('Create Notification template from blueprint - /notification-templates/:templateId/blueprint (POST)', async () => {
+describe('Create Notification template from blueprint - /blueprints/:templateId (POST)', async () => {
```

Comment:
I assume this is not a breaking change as is for Novu org.

Reply:
It will break changes for the pixel point project, i check already with Nevo he said it will be fine, they could update it on their side once we deploy.